gslib/tests/test_cat.py

```diff
@@ -162,3 +167,66 @@ def test_cat_encrypted_object(self):
       stdout = self.RunGsUtil(
           ['cat', '-r 1-3', suri(object_uri)], return_stdout=True)
       self.assertEqual(stdout, '123')
+
+
+class TestCatHelper(testcase.GsUtilUnitTestCase):
+  """Unit tests for cat helper."""
+
+  def test_cat_helper_runs_flush(self):
+    command_mock = mock.Mock()
+    cat_helper_mock = cat_helper.CatHelper(command_obj=command_mock)
+
+    object_contents = '0123456789'
+    bucket_uri = self.CreateBucket(bucket_name='bucket',
+                                   provider=self.default_provider)
+    obj = self.CreateObject(bucket_uri=bucket_uri,
+                            object_name='foo',
```

Comment:
`foo` and `foo1` isn't as descriptive as something like `foo1` and `foo2`. If you have `foo` without a number, people may assume there's just one of them.